op_crates/web/lib.rs

```diff
@@ -78,6 +79,28 @@ mod tests {
     });
   }
 
+  #[test]
+  fn test_event_error() {
+    run_in_task(|mut cx| {
+      let mut isolate = setup();
+      let result = isolate.execute("foo.js", "new Event()");
+      if let Err(error) = result {
+        let error_string = error.to_string();
+        // Test that the script specifier is a URL: `deno:<repo-relative path>`.
+        #[cfg(unix)]
+        assert!(error_string.starts_with("deno:op_crates/web/01_event.js"));
+        #[cfg(windows)]
+        assert!(error_string.starts_with("deno:op_crates\\web\\01_event.js"));
```

Comment:
thanks for the test in deno_web!

why use backslashes on windows?

Reply:
> why use backslashes on windows?

Only [special](https://url.spec.whatwg.org/#special-scheme) protocols (e.g. `file`) are actually specified to turn backslashes to forward slashes in the pathname. Our invented `deno:<os path>` protocol just excludes that logic by default. Do we want it?

Reply:
Hmm, even if it's not enforced by a URL standard it makes expected outputs a lot easier to write. We should do the replacement.